ptx/sec_deriv_interpret.ptx

```diff
@@ -1139,7 +1173,17 @@
             </pg-code>
             <statement>
               <image xml:id="img_02_02_ex_15">
-                <description></description>
+              <shortdescription> g is a downward parabola while f is a straight line that intersects g twice and the x-axis once.</shortdescription>
+                <description>
+                <p>
+                The graph of <m>g</m> is a a parabola whose focus is at <m>(2,-1)</m> in the 4-th quadrant.
+                The curve starts in the second qudrant close to the positive <m>y axis</m> then continues downward first 
+                intersects the <m>y-axis</m> then the <m>x-axis</m> close to the origin. Once it reaches the focus at <m>(2,-1)</m>,
+                it starts moving upwards again and eventually crosses the <m>x-axis</m> close to <m>x=4</m>.
```

Comment:
`<m>x</m> axis`, not `<m>x-axis</m>`. No hyphen before axis, and you do not want axis in math mode.